Dockerfile.build

```diff
@@ -7,7 +7,7 @@ RUN yum install -y rubygems libxml2 && \
     gem install asciidoctor && \
     yum clean all
 
-COPY user_guide/ user_guide/
-COPY common_content/ common_content
+COPY scripts/ scripts/
+COPY docs/ docs/
 
-WORKDIR user_guide
+# RUN scripts/validate_guides.sh
```

Comment:
Because validation fails on what I believe is a limitation of xmllint -- we use syntax that's valid Asciidoc, but it fails to validate. I need to investigate this. In the meantime, the validation isn't essential.